app/helpers/articles_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ArticlesHelper
```

Comment:
What is the purpose of this empty module?

Reply:
This module was created by default with the generate controller command:

`rails generate controller ArticleController Articles`

Reply:
Remember to be careful when using `generators`, they sometimes create files or code we don't need.

Do you think this `module` is necessary?

Reply:
At this moment no, so i must consider the command with the next parameters

` --no-helper`